blockchain/state/doc.go

```diff
@@ -18,5 +18,26 @@
 // This file is derived from core/state/statdb.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package state provides a caching layer atop the Klaytn state trie.
+/*
+Package state provides a caching layer atop the Klaytn state trie.
```

Comment:
what is mean 'atop"

Reply:
@nogang `atop` means "on the upper most level".

Reply:
https://endic.naver.com/enkrEntry.nhn?sLn=kr&entryId=887d516658e44492a76e0c9019016419 :)